src/schema/v2/GravityARImageType.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphQLNonNull, GraphQLObjectType, GraphQLString } from "graphql"
```

Comment:
I am not sure if this is the best approach. @mzikherman , maybe you can give me a recommendation. Gravity exposes `ARImageType`, which doesn’t exist on Metaphysics. I think ideally we shouldn’t use it and should migrate to a new image type native to Metaphysics. But I’m afraid that it will make the unstitching migration even more complex. That’s why I think we should introduce the data types that Gravity has but Metaphysics doesn’t (`ISO8601DateTime`, `ARImageType`, etc.). I cannot simply give them the same names; to avoid collisions, I prefix them with `Gravity` (and I think it’s worth marking them as deprecated from the start).

Reply:
Metaphysics already has `Image` that's used all over - can we use that? I think at some point we may not be able to produce an unstitched schema that's completely a no-op to clients. If the unstitched schema has breaking changes - the unstitched schema can maybe come in to MP as its 'own' schema (kind of like how 'Saved Search' stitched schema became 'Alert' unstitched schema), and then clients can switch to use it (while still using the unstitched schema in old Eigen installs for instance).

Then, after some period of time, we can simply remove the stitched schema as all clients would have migrated.

Is that a possibility? That feels less 'satisfying' since the stitched schema will hang around for longer, plus you'd need to update clients, but perhaps that's a more sane way (it kinda feels better to me than introducing all of these weird Gravity types here, but I haven't worked on the viewing room migration before so don't have the full mental model like you do).

Reply:
In that case, yes totally - let's roll with the easier compromise of adding `ARImageType`!